app-benchmarks/unigine-superposition/unigine-superposition-1.1.ebuild

```
@@ -0,0 +1,117 @@
+	opt/unigine-superposition-1.1/bin/qt/lib/libssl.so
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5QuickWidgets.so.5
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5QuickParticles.so.5
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5Qml.so.5
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5Quick.so.5
+	opt/unigine-superposition-1.1/bin/qt/lib/libicui18n.so.56
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5DBus.so.5
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5Network.so.5
+	opt/unigine-superposition-1.1/bin/qt/lib/libQt5Gui.so.5
+	opt/unigine-superposition-1.1/bin/qt/plugins/platforms/libqxcb.so
+	opt/unigine-superposition-1.1/bin/qt/plugins/xcbglintegrations/libqxcb-glx-integration.so
+	opt/unigine-superposition-1.1/bin/qt/plugins/xcbglintegrations/libqxcb-egl-integration.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtGraphicalEffects/libqtgraphicaleffectsplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtGraphicalEffects/private/libqtgraphicaleffectsprivate.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Window.2/libwindowplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Templates.2/libqtquicktemplates2plugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Controls.2/libqtquickcontrols2plugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Layouts/libqquicklayoutsplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Controls/libqtquickcontrolsplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Dialogs/libdialogplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick/Dialogs/Private/libdialogsprivateplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/QtQuick.2/libqtquick2plugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/Qt/labs/settings/libqmlsettingsplugin.so
+	opt/unigine-superposition-1.1/bin/qt/qml/Qt/labs/folderlistmodel/libqmlfolderlistmodelplugin.so
+	opt/unigine-superposition-1.1/bin/superposition_cli
+	opt/unigine-superposition-1.1/bin/libGPUMonitor_x64.so
+	opt/unigine-superposition-1.1/bin/libUnigine_x64.so
+	opt/unigine-superposition-1.1/bin/superposition
+	opt/unigine-superposition-1.1/bin/libopenvr_api.so
+	opt/unigine-superposition-1.1/bin/libopenal.so
+	opt/unigine-superposition-1.1/bin/libAppVive_x64.so
+	opt/unigine-superposition-1.1/bin/launcher
+	opt/unigine-superposition-1.1/Superposition
+"
+QA_FLAGS_IGNORED="${QA_PREBUILT}"
+
+src_unpack() {
+	sh "${DISTDIR}"/Unigine_Superposition-1.1.run --tar xfv --one-top-level=${P} || die
+}
+
+src_install() {
+	local res
+	for res in 16 24 32 48 64 128 256
+	do
+		newicon -s ${res} icons/superposition_icon_${res}.png Superposition.png
+	done
+
+	rm -rf icons {post,un}install.sh version bin/qt/lib/libQt5QuickTest.so.5 || die
+	# so.5 looks like unused lib https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=unigine-superposition#n76
+
+	chrpath -r '$ORIGIN/qt/lib' bin/launcher || die
+	# Against "scanelf: rpath_security_checks(): Security problem with relative DT_RPATH"
+
+	insinto /opt/${P}
```

```suggestion
	insinto /opt/${PN}
```
Is there a need to add the version to the path here? It'll force you to update the desktop file on each version bump so I'd just install into `/opt/${PN}`

---

Fixed.